applications/nrf_desktop/configuration/common/ble_adv_def.h

```diff
@@ -19,6 +19,11 @@ const struct {} ble_adv_def_include_once;
 
 static const struct bt_data ad_unbonded[] = {
 	BT_DATA_BYTES(BT_DATA_FLAGS, (BT_LE_AD_GENERAL | BT_LE_AD_NO_BREDR)),
+	/* nRF Desktop devices are assumed to have transmit power set to 0 dBm. */
```

Comment:
`... during advertising.`
For connections we explicitly modify the TX power so assumption would not be valid.